multiversx_sdk/core/transactions_factories/relayed_transactions_factory.py

```diff
@@ -72,6 +72,34 @@ def create_relayed_v2_transaction(self,
             options=inner_transaction.options
         )
 
+    def create_relayed_v3_transaction(self,
+                                      relayer_address: IAddress,
+                                      inner_transactions: Sequence[ITransaction]) -> Transaction:
+        if len(inner_transactions) == 0:
+            raise InvalidInnerTransactionError("The are no inner transactions")
+
+        gas_limit = self._config.min_gas_limit * len(inner_transactions)
```

Comment:
i think we need `50_000` more gas. Isn't it?
```
min_gas_limit + 50_000 * len(inner_transactions)
```

And to this we add the gasLimit for each relayed tx, which you've done bellow.

Reply:
changed